docs/client/operations.md

```diff
@@ -0,0 +1,228 @@
+
+AutoRest provides both synchronous and asynchronous method overloads for each service operation.
+Depending on your swagger definition, operations can be accessed through operation groups (TODO: link to swagger docs) on the client,
+or directly on the client.
+
+## Operation Group vs No Operation Group
+
+If your swagger defines an operation group for your operation (for example, in [this][operation_group_example] swagger, the operation `list`
+is part of operation group `application`), you would access the operation through the operation group getter on the client, `getApplications()`.
+This makes the call `client.getApplications().list()`.
+
+If there's no operation group, as in [this][mixin_example] case, you would access the operation directly from the client
+itself, i.e. `client.getDog()`.
+
+## Regular Operations
+
+### Sync Operations
+
+We will be using the [example swagger][pets_swagger] in our main docs repo. After [initializing][initializing] our client, we
+call our operation like this:
+
+```java
+package com.azure.pets;
+
+import com.azure.identity.DefaultAzureCredentialBuilder;
+import com.azure.pets.models.Dog;
+
+public static void main(String args[])
+{
+    PetsClient client = new PetsClientBuilder()
+        .credential(new DefaultAzureCredentialBuilder().build())
+        .buildClient();
+
+    Dog dog = client.getDog();
+}
+```
+
+### Async Operations
+
+When calling our async operations, we use our async client, which is suffixed with `AsyncClient` instead of `Client`. Our async operations
+return an [`Observable`][observable], so they need to be subscribed to. Following the [example above](#sync-operations Sync Operations),
+our call to `getDog` looks like this:
+
+```java
+package com.azure.pets;
+
+import com.azure.identity.DefaultAzureCredentialBuilder;
+import com.azure.pets.models.Dog;
+
+public static void main(String args[])
+{
+    PetsAsyncClient client = new PetsClientBuilder()
+        .credential(new DefaultAzureCredentialBuilder().build())
+        .buildAsyncClient();
+
+    client.getDog().subscribe(
+        d -> dog = d
+    );
+}
+```
+
+## Long Running Operations
+
+Long-running operations are operations which consist of an initial request sent to the service to start an operation, followed by polling the service at intervals to determine whether the operation has completed or failed, and if it has succeeded, to get the result.
+
+In concurrence with our [java guidelines][poller_guidelines], all of our long running operations are prefixed with `begin`, to signify the starting of the long running operation.
+
+For our example, we will use the long running operation generated from [this][example_swagger] swagger. Let's say we generated this swagger with namespace `com.azure.lro`.
+
+### Sync Long Running Operations
+
+Our sync long running operations return a [`SyncPoller`][lro_poller] polling object. Calling `.getFinalResult()` on this poller
```

Comment:
Right now, we probably do not yet support LRO in data-plane code generator.

Reply:
That's correct. We do not return a poller on any of the generated clients as the `x-ms-long-running-operation` does not contain enough information how the polling should behave for a data plane API.

Reply:
I see, I had my. suspicions but wanted to get some LRO docs out as well. I can remove them until you guys add LRO support, also lmk if you have questions about how python does it (basically we inspect the first response to figure out where to find the status / result)

Reply:
ARM part of LRO is more consistent and can be handled by status code and headers like `Azure-AsyncOperation` and `Location` (I guess this is what python does), and hence it only exists in `fluent` flavor as it is for ARM. I guess data-plane could be in different and not very consistent format.

Reply:
For now you can stash this section and put something like "The Java team is building the poller generation - will update when it's available."